.github/workflows/get_modular_resources.py

```diff
@@ -55,7 +55,6 @@ async def main():
                 }
                 insert_resp = typesense_client.collections['resources'].documents.upsert(
         json_m)
-                print(insert_resp)
```

Comment:
Please keep this - it's here for logging purposes for the github action, thanks!

Reply:
oops